src/main/scala/amailp/intellij/robot/psi/reference/LibraryToDefinitionReference.scala

```diff
@@ -0,0 +1,44 @@
+package amailp.intellij.robot.psi.reference
+
+import amailp.intellij.robot.psi.utils.ExtRobotPsiUtils
+import amailp.intellij.robot.psi.LibraryValue
+import com.intellij.codeInsight.lookup.{AutoCompletionPolicy, LookupElementBuilder}
+import com.intellij.psi.{PsiElement, PsiElementResolveResult, ResolveResult, PsiPolyVariantReferenceBase}
+import com.intellij.psi.util.QualifiedName
+import com.jetbrains.python.psi.PyFile
```

Comment:
The plugin should work even without the python plugin installed (for java users for example). In this case I guess they would get a ClassDefNotFound exception that is not what should happen. If you look at the moment the com.jetbrains.python.* classes are only loaded when the optional python plugin is present via an optional plugin desctriptor (META-INF/robot-python.xml)